### PR TITLE
Fix nested structs field names

### PR DESCRIPTION
**Pull request by kirr**

### CODE REVIEW

File: validator.go [hunk 1]
```diff
@@ -467,11 +470,18 @@ func parseName(tag string) string {
...
+	tags := strings.Split(tag, ",")
+	name := tags[0]
 	// if the field as be skipped in json, just return an empty string
 	if name == "-" {
 		return ""
 	}
+
+	for _, v := range tags {
+		if v == "inline" {
+			return ""
```
Comment (jameshartig) [inline validator.go]: This part seems unnecessary. If you specified `json:blah,inline` that's invalid and a linter should complain

**Review by jameshartig — COMMENTED** [review]
One small thing

File: validator.go (hunk 1 above)
Comment (jameshartig) [reply validator.go]: This would also break `json:"inline"`

Comment (kirr) [reply validator.go]: You are right. Thanks.

Also knew that
```
struct A {
  FieldName B `",inline"`
}
```
does not work in default go encoding; there is no special meaning for "inline" tag.

**Review by jameshartig — APPROVED** [review]

**Comment by jameshartig** [comment]
@kirr this is merged and included in v2.0.1

**Pull request merged**